docs/core/tools/dotnet-restore.md

```diff
@@ -43,7 +43,23 @@ For dependencies, you specify where the restored packages are placed during the
 
```

Comment:
you'd probably want to update the ms.date here no?

Reply:
I generally don't update the date unless the entire article was refreshed/validated. I only updated the subsection area and didn't really look a the entire article. I figured this will be updated when the CLI updates come in for 3.0